src/main/java/touro/chess/PawnPiece.java

```diff
@@ -49,9 +49,8 @@ public PawnPiece(Location location, PieceColor color) {
                 }
             }
 
-        }
+        } else if (super.getColor().toString().equals("White")) {
```

Comment:
Can you put both colors in one if statement? Like if (color = black and row = 1) or (color =  white and row = 6)? And I think isLegal can check for you if the piece can move there.